addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/discovery/MaxCubeBridgeDiscovery.java

```diff
@@ -44,9 +46,19 @@
 	private final static Logger logger = LoggerFactory.getLogger(MaxCubeBridgeDiscovery.class);
 
 	static boolean discoveryRunning = false;
-
+	
+	/** The refresh interval for discovery of MAX! Cubes */
+	private long refreshInterval = 20;
```

Comment:
@kaikreuzer It is definitely too short. Good catch. It was what I used for testing... I removed the 1 in front of it, as my suggested value was 120 seconds.

When I'm back in NL next week will fix this and look better to the comments wrt to the override.